CHANGELOG.md

```diff
@@ -1,3 +1,178 @@
+# 0.28.0 (2019-04-??)
+
+## Language changes
+
+- **(breaking-change)** Enum declaration members can no longer be separated by a comma, only by a newline or `;`. ([#7607](https://github.com/crystal-lang/crystal/pull/7607), [#7618](https://github.com/crystal-lang/crystal/pull/7618), thanks @asterite)
```

Comment:
This is not quite true: you can still use a comma but the formatter will change it to semicolon. The comma is deprecated, but you can still use it. Also, I'm not the one that implemented it.